app_backend/server.py

```diff
@@ -35,7 +45,7 @@ async def root():
         Response.status: int
             The status code for the request
 """
-@app.post("/new-friend/", status_code=status.HTTP_201_CREATED)
```

Comment:
why did the last backslash need to be removed? (just curious) 🧐

Reply:
With the backslash, it was causing 302 errors that rerouted on `post` requests. I honestly have no idea what that means or why, but the internet said to get rid of it and it worked lol